types/postman-sandbox/postman-sandbox-tests.ts

```diff
@@ -0,0 +1,50 @@
+let isString: string;
```

Comment:
It's better to use [`$ExpectType`](https://github.com/Microsoft/dtslint#write-tests) than this style anyway.